salesforce/latest/using/community.md

```diff
@@ -0,0 +1,139 @@
+
+12. Select the **Salesforce Community Name** you would like to share with and the **Users**. Click **Save**.
+
+      ![lightning-externally-select]({% link salesforce/images/lightning-externally-select.png %})
+
+You will see a **Salesforce Community Member Access** message. The result of the performed actions is that when the Community Member logs into Salesforce they will see the case in their **All Open Cases** list.
+
+## Salesforce Classic
+
+1. See [Create an Experience Cloud Site pg. 51](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to Community Members Salesforce Classic](#share-content-externally-to-community-members-salesforce-classic)
+
+## Share content externally to Community Members Salesforce Classic
+
+1. Log in to Salesforce.
+
+2. Click **Alfresco Repository** and Log in as an administrator to Alfresco Share in the Canvas app.
+
+      ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
+
+3. Click **Cases** and click the **Create New** drop down list and then select **Case**.
+
+      ![classic-create-case]({% link salesforce/images/classic-create-case.png %})
+
+4. Select the required **Case Origin** and any other information needed for the case. Click **Save**.
+
+      ![classic-new-case]({% link salesforce/images/classic-new-case.png %})
+
+5. Click the **Sharing** button above the case.
+
+      ![classic-sharing]({% link salesforce/images/classic-sharing.png %})
+
+6. Click the **Add** button.
+
+      ![classic-sharing-add]({% link salesforce/images/classic-sharing-add.png %})
+
+7. Search **Public Groups** and add **All Customer Portal Users** to the **Share With** column. Click **Save**.
+
+      ![classic-sharing-groups]({% link salesforce/images/classic-sharing-groups.png %})
+
+8. Click the **Create** drop down list and select **Folder**.
+
+      ![classic-create-folder]({% link salesforce/images/classic-create-folder.png %})
+
+9. Enter a name for the folder and click **Create**.
+
+      ![classic-name-folder]({% link salesforce/images/classic-name-folder.png %})
+
+10. Click the **Actions** button and select **Share Externally**.
+
+      ![classic-share-externally]({% link salesforce/images/classic-share-externally.png %})
+
+11. Select the **Salesforce Community Name** you would like to share with and the **Users**. Click **Save**.
+
+      ![lightning-externally-select]({% link salesforce/images/lightning-externally-select.png %})
+
+You will see a **Salesforce Community Member Access** message. The result of the performed actions is that when the Community Member logs into Salesforce they will see the case in their **All Open Cases** list.
+
+## Retrieving shared cases
```

Comment:
Minor update, maybe change to "Community Member accessing shared content"

Reply:
Changed cheers